pandas/core/computation/eval.py

```diff
@@ -148,7 +147,7 @@ def _check_for_locals(expr, stack_level, parser):
 
 def eval(expr, parser='pandas', engine=None, truediv=True,
          local_dict=None, global_dict=None, resolvers=(), level=0,
-         target=None, inplace=None):
+         target=None, inplace=False):
```

Comment:
Defer to @jreback, @jorisvandenbossche, but I am still of the opinion that API would be cleaner if `inplace` was only allowed with a `target` and raised otherwise.  `pd.eval('1 + 2', inplace=True) == None` is surprising.